src/main/java/cdw/springtraining/gatekeeper/entites/GateKeeper.java

```diff
@@ -0,0 +1,57 @@
+package cdw.springtraining.gatekeeper.entites;
+
+import jakarta.persistence.*;
```

Comment:
import only needed package and avoid importing all packages

Reply:
removed all unused packages